include/SFML/Graphics/Font.hpp

```diff
@@ -427,7 +427,7 @@ class SFML_GRAPHICS_API Font
     mutable PageTable            m_pages;          //!< Table containing the glyphs pages by character size
     mutable std::vector<std::uint8_t> m_pixelBuffer; //!< Pixel buffer holding a glyph's pixels before being written to the texture
 #ifdef SFML_SYSTEM_ANDROID
-    std::unique_ptr<priv::ResourceStream> m_stream; //!< Asset file streamer (if loaded from file)
+    std::shared_ptr<priv::ResourceStream> m_stream; //!< Asset file streamer (if loaded from file)
```

Comment:
My first impression was to be wary of this introduction of shared ownership but a few lines above this we already have a shared pointer in `m_fontHandles` so it seems like `sf::Font` already sets the precedent that some data has shared ownership.